### PR TITLE
Correct message about k3s.yaml

### LINKED ISSUE #1471: Could not read message is misleading
The text says `could not read`:
https://github.com/rancher-sandbox/rancher-desktop/blob/6c34341ffe5fb56b59c940c7db639ad99e08116d/src/k8s-engine/lima.ts#L1391

But the error is **could not _list_**:
https://github.com/rancher-sandbox/rancher-desktop/blob/6c34341ffe5fb56b59c940c7db639ad99e08116d/src/k8s-engine/lima.ts#L1383-L1384

The user who will run this command doesn't have permission to read the file (it's owned by `root` and has permissions `0600`).

(Spotted while debugging a problem relating to `~/.kube` configuration on the host that resulted in an error involving `kubectl` with this being one of the last outputs before the error. Using versions 1.0.0 and 1.0.1 on macOS Intel.)

### PR DESCRIPTION
**Pull request by jsoref**
Fixes #1471

### CODE REVIEW

File: src/k8s-engine/lima.ts [hunk 1]
```diff
@@ -1392,7 +1392,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
                   { env: this.limaEnv, stdio: 'ignore' });
                 break;
               } catch (ex) {
-                console.log('Could not read k3s.yaml; retrying...');
+                console.log('Configuration k3s.yaml not present; will check again...');
```
Comment (jandubois) [inline src/k8s-engine/lima.ts:1395]: Change seems very subtle, but ok.

However, I would prefer if the log message includes the full path to the missing file:

```suggestion
                console.log('Configuration file /etc/rancher/k3s/k3s.yaml not present; will check again...');
```

Comment (jsoref) [reply src/k8s-engine/lima.ts:1395]: Should I specify `in lima vm`?

Comment (jandubois) [reply src/k8s-engine/lima.ts:1395]: > Should I specify `in lima vm`?

sure

**Review by jandubois — APPROVED** [review]
You left out the "file" in "Configuration file /etc/...", but good enough for me

**Pull request merged**